kasa/cli.py

```diff
@@ -51,6 +51,8 @@ def wrapper(message=None, *args, **kwargs):
 
 pass_dev = click.make_pass_decorator(SmartDevice)
 
+echo = _do_echo
```

Comment:
Move the assignment directly after the definitions, and add a short one-line comment to describe why it's done like this. Then I think it's good to go, fixing it properly (if worth it) belongs to another PR and another time..

Reply:
Done